src/fprime/util/cli.py

```diff
@@ -16,7 +16,7 @@
 from fprime.fbuild.target import Target
 from fprime.fpp.cli import add_fpp_parsers
 from fprime.util.build_helper import load_build
-from fprime.util.commands import run_code_format, run_hash_to_file, run_info, run_new
+from fprime.util.commands import run_code_format, run_hash_to_file, run_info, run_new, run_sysinfo
 from fprime.util.help_text import HelpText
 from fprime.fpp.visualize import add_fpp_viz_parsers
 from fprime.fpp.impl import add_fpp_impl_parsers
```

Comment:
In this file, you should be able to add the new command to the [skip_build_loading()](https://github.com/0x48piraj/fprime-tools/blob/863bde64a8965b881d1fdddd4729437e10d9d4b6/src/fprime/util/cli.py) function 

This will skip trying to load the Build object (see [here](https://github.com/0x48piraj/fprime-tools/blob/863bde64a8965b881d1fdddd4729437e10d9d4b6/src/fprime/util/cli.py#L30-L34)), and we'll be able to run this command form anywhere, no need to be in an F´ project

The following should do the trick
```
    if parsed.command == "sysinfo":
        return True
```